### ISSUE TITLE
Windows 102.2 -> 103 error: wrong-type-argument in package--read-archive-file 

### ISSUE THREAD

**Issue by bnns** [syl20bnr/spacemacs#2179]
Got this error when I tried to switch to the develop branch earlier, now also in master. Also happens after I remove .emacs.d and re-clone. Windows 8 running emacs 25.

From the --debug-init of my emacs:

```  
Debugger entered--Lisp error: (wrong-type-argument listp closed)
  package--read-archive-file("archives/gnu/archive-contents")
  package-read-archive-contents("gnu")
  package-read-all-archive-contents()
  package--update-downloads-in-progress(("gnu" . "https://elpa.gnu.org/packages/"))
  package--check-signature("https://elpa.gnu.org/packages/" "archive-contents"
  ...
  spacemacs/load-or-install-package(dash)
  spacemacs/load-theme(solarized-light)
  spacemacs/init()
```

**Comment by justbur** [comment]
There are a lot of changes in package.el for emacs25. See #2148 and #2045 as two examples that I've run into. I've given up on using the spacemacs update mechanism for now when I'm using emacs25

**Comment by bnns** [comment]
But this happens even on a clean install of spacemacs. Should I revert to 102.2?

**Comment by justbur** [comment]
I've been on 103 with w8 and emacs25 for weeks now, so no you don't need to
revert. I was saying that errors with emacs25 popup here and there,
especially with the upgrade mechanism and package.el. If you want to use
25, you'll probably have to work around them, since it's not really fully
supported by spacemacs. emacs24 should be much more stable in that regard.

Maybe it's a random thing with the gnu repository not being available (I'm
looking at your message again). Did you try it again later?

**Comment by bnns** [comment]
Yes, now it works! I'm not sure what happened - perhaps it was the gnu repository being down, but I got the error on at least 5 different occasions before when I tried to switch to `develop` branch.

I'm speculating that the cause was committing some stuff in my previous local repos that I shouldn't have. For others having this problem, setting autocrlf in git might help:
```
git config --global core.autocrlf true
```
Anyway, thanks a lot for the suggestions.
